plugins/backend/backstage-aws-backend/src/plugin.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  coreServices,
+  createBackendPlugin,
+} from '@backstage/backend-plugin-api';
+import { createRouter } from './service/router';
+
+export const AWSBackendPlugin = createBackendPlugin({
+  pluginId: 'aws',
+  register(env) {
+    env.registerInit({
+      deps: {
+        http: coreServices.httpRouter,
+        logger: coreServices.logger,
+        config: coreServices.rootConfig,
+      },
+      async init({ http, logger, config }) {
+        logger.info('AWSBackendPlugin is initializing');
+        http.use(
+          await createRouter({
+            logger,
+            config,
```

Comment:
MINOR: Just sorting keys.

```suggestion
      deps: {
        config: coreServices.rootConfig,
        http: coreServices.httpRouter,
        logger: coreServices.logger,
      },
      async init({ config, http, logger }) {
        logger.info('AWSBackendPlugin is initializing');
        http.use(
          await createRouter({
            config,
            logger,
```